homeassistant/scripts/check_config.py

```diff
@@ -0,0 +1,172 @@
+"""Script to ensure a configuration file exists."""
+import argparse
+import os
+import re
+from glob import glob
+import logging
+from typing import List
+import pprint
+from unittest.mock import patch
+from textwrap import fill
+from collections import OrderedDict
+
+import homeassistant.bootstrap as bootstrap
+import homeassistant.config as config_util
+# import homeassistant.util.yaml as yaml_util
+import homeassistant.loader as loader
+import homeassistant.util.yaml as yaml
+
+REQUIREMENTS = ['colorlog>2.1<3', 'colorama<=1']
```

Comment:
`colorama` provides formatting of non-log strings and `colorlog` is used by the HASS logger

Maybe I can do my prints with `colorlog`, but to cater for Windows users, `colorlog` depends on `colorama` to enable colours in windows. Their documentation mentions you should depend on `colorlog[windows]` for auto install, but that does not seem to work.